snuba/utils/metrics/backends/datadog.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional, Sequence, Union
+
+from datadog import DogStatsd
+
+from snuba.utils.metrics import Tags
+from snuba.utils.metrics.backends.abstract import MetricsBackend
+
+
+class DatadogMetricsBackend(MetricsBackend):
```

Comment:
Only the gauge is covered in this class. This should probably have some sort of integration test.